sdk/lib/core/map.dart

```diff
@@ -226,12 +226,10 @@ abstract class Map<K, V> {
   /**
    * Returns the value for the given [key] or null if [key] is not in the map.
    *
-   * Some maps allows keys to have `null` as a value,
-   * For those maps, a lookup using this operator does cannot be used to
-   * distinguish between a key not being in the map, and the key having a null
-   * value.
-   * Methods like [containsKey] or [putIfAbsent] can be use if the distinction
-   * is important.
+   * Some maps allows keys to have `null` as a value. For those maps, a lookup
+   * using this operator cannot be used to distinguish between a key not being
+   * in the map and the key having a null value. Methods like [containsKey] or
```

Comment:
null -> `null`

Reply:
Than you for this clean-up. That was some very badly edited doc we had there! :)